CMakeLists.txt

```
@@ -1,35 +1,126 @@
-cmake_minimum_required(VERSION 3.17...3.22)
+cmake_minimum_required(VERSION 3.17...3.26)
+
+project(pyopencl LANGUAGES CXX)
```

```suggestion
project(pyopencl LANGUAGES CXX VERSION ${SKBUILD_PROJECT_VERSION})
```
? I think `PROJECT_VERSION` is used below and it would be empty otherwise.

---

Great catch, thanks!